install_sct

```diff
@@ -45,6 +45,38 @@ function die() {
   exit 1
 }
 
+function fetch_os_type() {
+  # Fetch OS type
```

Comment:
it is a good habit to add a header in functions that indicates what are the outputs or updated variables (in this case `OS`).